tests/pkcli/admin_test.py

```diff
@@ -18,27 +18,32 @@ def test_purge_users(monkeypatch):
     srunit.init_auth_db(sim_types='myapp')
 
     from sirepo.pkcli import admin
+    from sirepo import auth
     from sirepo import simulation_db
-    from sirepo import auth_db
+    from sirepo import srtime
     import datetime
 
-    res = admin.purge_users(days=1, confirm=False)
+    days = 1
+    adjusted_time = days + 10
+
+    res = admin.purge_guest_users(days=days, confirm=False)
     pkeq([], res, '{}: no old users so empty')
+
     g = simulation_db.user_dir_name('*')
     dirs = list(pkio.sorted_glob(g))
     pkeq(1, len(dirs), '{}: expecting exactly one user dir', g)
+
     uid = dirs[0].basename
-    #TODO(robnagler) really want the db to be created, but need
-    #  a test oauth class.
-    monkeypatch.setattr(auth_db, 'all_uids', lambda: [uid])
-    for f in pkio.walk_tree(dirs[0]):
-        f.setmtime(f.mtime() - 86400 * 2)
-    res = admin.purge_users(days=1, confirm=False)
-    pkeq([], res, '{}: all users registered so no deletes')
-    monkeypatch.setattr(auth_db, 'all_uids', lambda: [])
-    res = admin.purge_users(days=1, confirm=False)
-    pkeq(dirs, res, '{}: no users registered so one delete', res)
-    pkok(dirs[0].check(dir=True), '{}: nothing deleted', res)
-    res = admin.purge_users(days=1, confirm=True)
-    pkeq(dirs, res, '{}: no users registered so one delete', res)
+    srtime.adjust_time(adjusted_time) 
+
+    monkeypatch.setattr(auth, 'guest_uids', lambda: [])
+    res = admin.purge_guest_users(days=days, confirm=False)
+    pkeq([], res, '{}: no guest users so no deletes')
+
+    monkeypatch.setattr(auth, 'guest_uids', lambda: [uid])
```

Comment:
I don't think you need to patch this any more. I think guest_uids will work, because auth_db is initialized. You might try it.